teku/src/main/java/tech/pegasys/teku/cli/BeaconNodeCommand.java

```diff
@@ -134,50 +133,65 @@ private static class ConfigFileCommand {
       scope = ScopeType.INHERIT)
   private File configFile;
 
-  @Mixin(name = "Network")
-  private Eth2NetworkOptions eth2NetworkOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Network%n", headingKey = "Network")
+  private Eth2NetworkOptions eth2NetworkOptions = new Eth2NetworkOptions();
 
-  @Mixin(name = "P2P")
-  private P2POptions p2POptions;
+  @CommandLine.ArgGroup(validate = false, heading = "P2P%n", headingKey = "P2P")
+  private P2POptions p2POptions = new P2POptions();
 
-  @Mixin(name = "Interop")
-  private InteropOptions interopOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Validator%n", headingKey = "Validator")
+  private ValidatorOptions validatorOptions = new ValidatorOptions();
 
-  @Mixin(name = "Validator")
-  private ValidatorOptions validatorOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Deposit%n", headingKey = "Deposit")
+  private DepositOptions depositOptions = new DepositOptions();
 
-  @Mixin(name = "Deposit")
-  private DepositOptions depositOptions;
+  @CommandLine.ArgGroup(
+      validate = false,
+      heading = "Execution Layer%n",
+      headingKey = "Execution Layer")
+  private ExecutionLayerOptions executionLayerOptions = new ExecutionLayerOptions();
 
-  @Mixin(name = "Execution Layer")
-  private ExecutionLayerOptions executionLayerOptions;
-
-  @Mixin(name = "Logging")
+  @CommandLine.ArgGroup(validate = false, heading = "Logging%n", headingKey = "Logging")
   private final LoggingOptions loggingOptions = new LoggingOptions();
 
-  @Mixin(name = "Metrics")
-  private MetricsOptions metricsOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Metrics%n", headingKey = "Metrics")
+  private MetricsOptions metricsOptions = new MetricsOptions();
```

Comment:
Let's move these to be the last two arg groups.  The help text seems to print things in the order defined here.